### ISSUE TITLE
Event Start and End Time different on Phone vs. Desktop

### ISSUE THREAD

**Issue by skemp353** [sapmentors/SITRegParticipantList#2]
For some reason the start and end time of an event is displayed differently depending on whether you are viewing it on the desktop or via a mobile device.

On my Mac via Chrome (Start time 17:00 end time 19:00):

![img_4797](https://cloud.githubusercontent.com/assets/6735410/19503482/9f19ccac-95ff-11e6-8a93-ba96adf76d6f.JPG)

Same event on my iPhone

![img_4796](https://cloud.githubusercontent.com/assets/6735410/19503474/8d213f3a-95ff-11e6-82a9-61a82ba6d406.PNG)

At this stage I really can't guess why!

The correct times are 17:00 and 19:00, not sure why it's showing one hour different on the phone.

**Comment by gregorwolf** [comment]
I would guess there is a different timezone setting on your phone. I've just tested on my computer and the times where adjusted after a refresh of the app when I've changed the timezone setting:

<img width="708" alt="screen shot 2016-10-19 at 23 56 04" src="https://cloud.githubusercontent.com/assets/2639535/19539083/cc434ff2-9657-11e6-9266-bf797e8940b7.png">

<img width="850" alt="screen shot 2016-10-20 at 08 55 22" src="https://cloud.githubusercontent.com/assets/2639535/19539084/cc4798b4-9657-11e6-97d8-763b812cefe8.png">

**Comment by skemp353** [comment]
It is strange. My phone automatically adjusts and sets the time. It says Time Zone = Sydney and shows me the correct time, but shows me one hour later for the start and end time of the event. Maybe it is somehow related to daylight savings settings. I need to investigate further.

**Comment by gregorwolf** [comment]
I think the main reason is that the times are an extra field and because only the time can be maintained, the date is 1970-01-01. Jan already suggested a fix https://github.com/sapmentors/SITreg/issues/21

**Comment by skemp353** [comment]
Yes, I agree with @jpenninkhof have a start date and time and and end date and time.
